zenodo/config.py

```diff
@@ -585,6 +584,7 @@ def _(x):
 SECURITY_PASSWORD_SALT = "CHANGE_ME"
 SECURITY_REMEMBER_SALT = "CHANGE_ME"
 SECURITY_RESET_SALT = "CHANGE_ME"
+SECURITY_PASSWORD_SCHEMES = ['plaintext', 'pbkdf2_sha512']
```

Comment:
But that might because the users are created with a plaintext password initially?